position.py

```diff
@@ -0,0 +1,21 @@
+from enum import Enum
+import wpilib
+
+class EPosition(Enum):

```

Comment:
Since we are just using integers do we still need the enum?

Reply:
If you look above, I've suggested how we might use the EPosition instead of raw ints. @anisaa2326 @DarthCooper let me know if this makes sense.